modules/testcontainers/src/main/kotlin/jp/co/soramitsu/iroha2/testcontainers/IrohaContainer.kt

```diff
@@ -18,34 +11,33 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.time.Duration
 import java.util.UUID.randomUUID
-import java.util.function.Consumer
 import kotlin.io.path.absolute
 import kotlin.io.path.createTempFile
 
 open class IrohaContainer(
-    private val networkToJoin: Network = newNetwork(),
-    private val logConsumer: Consumer<OutputFrame>? = Slf4jLogConsumer(getLogger(IrohaContainer::class.java)),
     imageTag: String = DEFAULT_IMAGE_TAG,
```

Comment:
I think we should make `imageTag` part of the configuration too. Something like that

```kotlin
open class IrohaContainer: GenericContainer<IrohaContainer> {

    private val config: IrohaConfig

    constructor(config: IrohaConfig.() -> Unit = {}) : this(IrohaConfig().apply(config))

    constructor(config: IrohaConfig) : super(DockerImageName.parse(config.imageName)) {
        this.config = config
        this.withNetwork(config.networkToJoin)
```